README.md

```diff
@@ -59,3 +59,13 @@ The analysis is split in three notebooks:
  - [1-Expression-VIP.ipynb](1-Expression-VIP.ipynb): Run the baseline analysis using L1000 profiling.
  - [2-Cell-Morphology-VIP.ipynb](2-Cell-Morphology-VIP.ipynb): Run the Cell Morphology VIP method.
  - [3-Aggregation-plots.ipynb](3-Aggregation-plots.ipynb): Create the plots summarizing results.
+
+## Notes about the dataset
+
+From the paper: 
+> An additional 158 constructs are included in the dataset, representing alleles that inadvertently had double mutations, or were similar to the ones selected except in the presence of an epitope tag. These additional alleles have been included in the dataset for completeness.
```

Comment:
I suggested to change this from 158 to 86. That's the number of dropped alleles (out of the 412 total), ending up with 326 (325 + EMPTY). We should update this accordingly.